config/locales/client.en.yml

```diff
@@ -1,5 +1,8 @@
 en:
   js:
-    code_bytes:
     composer:
       codebyte: "Create a Codebyte"
+    modal:
+      title: "Please select a language"
+      content: "Your post includes one or more Codebytes that has no language selected. You can select a language for each Codebyte in the preview panel on the right."
```

Comment:
Nice surprise. 
Just a note to check in with product this copy. Feel free to resolve this thread if this was already signed off on.

<img width="701" alt="Screen Shot 2021-06-14 at 12 11 43 PM" src="https://user-images.githubusercontent.com/17210163/121924561-04dea800-cd0a-11eb-96ee-85fa54d11a6e.png">

Reply:
Yep, got the copy from Li.